portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,113 @@
-#content {
+/* #content {
```

Comment:
same here

Reply:
Done! Thanks! I removed all commented unused code in css style.